scripts/github_api

```diff
@@ -71,6 +71,40 @@ def _print_output(output):
         print '%s="%s"' % (k, v)
 
 
+class Status:
+    """
+    https://developer.github.com/v3/repos/statuses/
+    """
+    def __init__(self, repo, reference):
+        """
+        Args:
+            repo (str): target repository path. Format:
+                <repo owner>/<repo>
+            reference (str): git reference. Either a sha1 or a
+                pull-request number
```

Comment:
- I'd prefer to not use a dash ("-") in "pull-request"
- A pull request number is not a git reference. Maybe you meant "a git reference (sha1 commit ID, tag name or branch name) or a GitHub pull request number"
- Do we need to accept a git reference? We'll always want to query and update a pull request status and we'll have its number